src/shared/stack_view_action.rs

```diff
@@ -0,0 +1,8 @@
+use makepad_widgets::*;
+
+#[derive(Clone, WidgetAction)]
+pub enum StackViewAction {
+    None,
+    ShowAddContact,
+    ShowMoments
+}
```

Comment:
NIT, for the future if we ever need it:
Since we're repeating `Show` and If we were to have actions other than show, I'd probably split this

```
pub enum StackViewAction {
    None,
    Show(View)
}

pub enum View {
  AddContact,
  Moments,
  ...
}
```
right now we only have a couple so not really relevant 🙌🏽